ci/prepare-unity.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source ci/includes/pinned-tools.sh
+source ci/includes/profiling.sh
+
+function isLinux() {
+  [[ "$(uname -s)" == "Linux" ]];
+}
+
+function isMacOS() {
+  [[ "$(uname -s)" == "Darwin" ]];
+}
+
+function isWindows() {
+  ! ( isLinux || isMacOS );
+}
+
+# Only run inside TeamCity
+if [ -z "${TEAMCITY_CAPTURE_ENV+x}" ]; then
+    exit 0
```

Comment:
add some kind of message here to signal to the user that this can only be run in TC /  why should it only be run on TC? wouldn't that also make it easier for us, if we can just run this script to get the new version?

Reply:
I really don't want this to install unity for our clients, hence this only runs for CI to avoid some manual work until the new CI is here.